test/clj/cider/nrepl/middleware/util/instrument_test.clj

```diff
@@ -138,6 +138,14 @@
          '#{[(def foo "foo doc" (bp (bar) {:coor [3]} (bar))) []]
             [(bar) [3]]})))
 
+(deftest instrument-dot-test
+  (is (= (t/breakpoint-tester '(. clojure.lang.Numbers add 1 x))
```

Comment:
Might be a good idea to wrap the tests in `testing` blocks with some explanations of why those cases are different.